docs/update_petro_real.rst

```diff
@@ -0,0 +1,185 @@
+in ERT in assisted histroy matching and therefore not included here,
+but still used in the RMS workflow with the prior values.
+
+The original realizations created for PHIT and KLOGH are conditioned
+to facies. In a workflow where we want to use both facies by applying
+the Adaptive PluriGaussian Simulation method (APS) and PHIT and KLOG
+for some selected facies as field parameters to be updated by ERT,
+we will need the separate PHIT and KLOGH realizations for the selected
+facies.
+
+The steps will be:
+
+    * Use original petrophysical job to create initial version of a realization of
+      PHIT, KLOGH, VSH and VPHYL.
+    * Use separate petrophysical jobs to create separate version of realizations of
+      Floodplain_PHIT, Floodplain_KLOGH and so on.
+    * Use the script *update_petro_real* to copy the values for PHIT and KLOGH from
+      the individual realizations per facies into the initial version of PHIT and KLOG
+      by overwriting the values in the original version. In this process,
+      the facies realization is used as a filter to select which grid cell values
+      to get from the individual PHIT and KLOGH parameters that belongs to the
+      various facies.
+
+When running ERT iteration 0 which creates the initial ensemble, this looks a bit unnecessary,
+since the initial PHIT and KLOGH already has taken facies into account. But, when running
+ERT iteration larger than 0 (after ERT has updated the fields Floodplain_PHIT,
+Floodplain_KLOGH and so on), then updated versions of the field parameters are imported
+and updated version of facies realization is used to update the PHIT and KLOGH parameters
+from the imported field parameters (Floodplain_PHIT, Floodplain_KLOGH ...).
+
+Also for iteration 0, to ensure consistency the same procedure
+(copy from the field parameters Floodplain_PHIT, Floodplain_KLOG,..) is
+applied since this ensure consistent handling of the realizations.
+
+Example of RMS python job running this module to combine the field parameters
+-------------------------------------------------------------------------------
+
+The python job below will call the *update_petro_real* function that does the
+combination of the field parameters into one petrophysical realization.
+This job depends on a small config file ( the same config file that is used by the
+function *generate_petro_jobs*  from fmu.tools.rms module). Here the python job gets
+the grid name and which of the petrophysical properties for which facies
+to be updated as field parameters in ERT. The script also depends on the facies table
+(which facies code and name belongs together).
+This can be fetched from the *global_variables.yml* file as shown in the example below.
+
+An alternative is to add a key *USED_PETRO_FIELDS*  in global_master_config.yml
+and get the petro parameters per facies per zone from this keyword instead.
+
+.. code-block:: python
+
+    from fmu.tools.rms.update_petro_real import update_petro_real
```

Comment:
Perhaps adapt the __init__ file here so we can use:

`from fmu.tools.rms import update_petro_real`

Reply:
fixed